packages/thumbprint-atomic/src/packages/cursor.scss

```diff
@@ -0,0 +1,56 @@
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+@import '~@thumbtack/thumbprint-scss/mixins';
+
+.cursor-default {
+    cursor: default !important;
+}
+
+.cursor-auto {
+    cursor: auto !important;
+}
+
+.cursor-pointer {
+    cursor: pointer !important;
+}
+
+@include tp-respond-above($tp-breakpoint__small) {
+    .s_cursor-default {
+        cursor: default !important;
+    }
+
+    .s_cursor-auto {
+        cursor: auto !important;
+    }
+
+    .s_cursor-pointer {
+        cursor: pointer !important;
+    }
+}
+
+@include tp-respond-above($tp-breakpoint__medium) {
+    .m_cursor-default {
+        cursor: default !important;
+    }
+
+    .m_cursor-auto {
+        cursor: auto !important;
+    }
+
+    .m_cursor-pointer {
+        cursor: pointer !important;
+    }
+}
+
+@include tp-respond-above($tp-breakpoint__large) {
+    .l_cursor-default {
+        cursor: default !important;
+    }
+
+    .l_cursor-auto {
+        cursor: auto !important;
+    }
+
+    .l_cursor-pointer {
+        cursor: pointer !important;
+    }
+}
```

Comment:
Are there cases where we'd want to have the cursor behave differently depending on a breakpoint? If not, I think it's best to not include these. Keeps the file size small!

Reply:
I think there could be scenarios when people only want to apply pointer cursor in small screen, which means a combo of "s_cursor-default" + "cursor-pointer" would be needed

Reply:
Hm, do you have examples of scenarios? I can't really think of any.

Reply:
the example will depend on the design requirement. I'm proposing this because i've seen similar cases when using other atomic classes, such as "flex" + "s_db" so only small screen has flexbox. basically it's the scenarios when you want to make special style only **below** instead of **above** a breakpoint

Reply:
Yeah, the responsive classes are definitely a helpful pattern. But we don't add them for all classes because it makes more sense for some than it does for others. Those layout classes are commonly used because layouts always change when designing responsive screens. 

We don't add responsive classes to colors though since it's a lot less likely that colors change depending on the browser viewport. Cursor changing would be even less likely since the cursors don't exist on touch devices.

(Forgot to respond yesterday. Sorry for the delay!)

Reply:
got it. that makes sense!